### PR TITLE
Update docker-compose.yml

### PR DESCRIPTION
**Pull request by YoKv**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix

___
### Bug fix
- config folder error

### CODE REVIEW

File: docker/docker-compose.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ services:
       - 11800:11800
       - 12800:12800
     volumes:
-      - ./config:/apache-skywalking-apm-incubating/config:ro
+      - ./config:/skywalking/config:ro
```
Comment (hanahmily) [inline docker/docker-compose.yml]: should be `./config:/apache-skywalking-apm-incubating-bin/config:ro` according to [OAP Dockerfile](https://github.com/apache/incubator-skywalking/blob/master/docker/oap/Dockerfile)

**Comment by YoKv** [comment]
@hanahmily
```
# docker pull skywalking/oap
Using default tag: latest
latest: Pulling from skywalking/oap
Digest: sha256:1e3a411e7a3b02738e9651e1e323510c35a568e804f1bfc502f9600e9ff8ceb0
Status: Image is up to date for skywalking/oap:latest
```

```
# docker run -d --name sw skywalking/oap
# docker exec -it sw ls /
bin           lib           root          srv           usr
dev           media         run           sys           var
etc           mnt           sbin          tmp
home          proc          skywalking    trace-buffer
```

**Comment by wu-sheng** [comment]
`skywalking/oap` is official release for stable version. `docker/docker-compose.yml` is related to `/docker/oap/Dockerfile`. I think @hanahmily give the right new name.

**Comment by YoKv** [comment]
@wu-sheng ok i know,thank you anyway.

**Comment by wu-sheng** [comment]
Any update?

**Comment by YoKv** [comment]
> Any update?

no updates,can i close this pr

**Pull request closed without merging**